app/lib/forem_stats_drivers/datadog_driver.rb

```diff
@@ -0,0 +1,19 @@
+module ForemStatsDrivers
+  class DatadogDriver
+    include ActsAsForemStatsDriver
+    setup_driver do
```

Comment:
Nice, glad to see my suggestion actually worked, it's really nice how declarative this feels!